### PR TITLE
Add logic for testing of specific packages

### PR DESCRIPTION
**Pull request by bradsawadye**
Logic for specifying the package to be tested has been added. The logic also caters for custom package testing.

DIO-52 DIO-56

### CODE REVIEW

**Comment by citizenrich** [comment]
This is great. I wonder if there is interest to include running the tests on the core package in GitHub Actions in this repo?

File: README.md [hunk 1]
```diff
@@ -131,16 +131,39 @@ Multiple custom packages can be chained together as follows:
...
-For docker instances, update the `.env.local` file with the instances' host urls and ports, and then run
+#### Docker-Compose
+
+Run the following for the default tests:
+
+```sh
+yarn test:local <PACKAGE_IDs>
+```
+
+If you want to make changes to the tests, you can run your changes without rebuilding anyting by using the *dev* version of the command:
```
Comment (bradsawadye) [inline README.md:142]:
```suggestion
If you want to make changes to the tests, you can run your changes without rebuilding anything by using the *dev* version of the command:
```

File: README.md [hunk 2]
```diff
@@ -131,16 +131,39 @@ Multiple custom packages can be chained together as follows:
...
+Update the `.env.remote` file with the instances' host urls and ports.
+Then update the file path in the [`package.json`](./package.json) file on line 22 in the scripts section.
+This file path needs to reference your `.env.remote` file to volume in your updates.
+Finally, run the following command for the default tests:
+
+```sh
+yarn test:remote <PACKAGE_IDs>
+```
+
+If you want to make changes to the tests, you can run your changes without rebuilding anyting by using the *dev* version of the command:
```
Comment (bradsawadye) [inline README.md:161]:
```suggestion
If you want to make changes to the tests, you can run your changes without rebuilding anything by using the *dev* version of the command:
```

**Comment by MattyJ007** [comment]
@citizenrich I'd love to set that up but I've never actually used Actions before (just skimmed the options). Would we be able to spin up a powerful enough server for testing all this infrastructure? 
We'd need a large amount of RAM (~16GB) and a decent chip (Do we have the budget for this?)

In the Action flows they all say spin up **a docker container** - would we be able to add multiple?

**Comment by danfuterman** [comment]
It'd be great to look into Github Actions and see how we could use that going forward (i.e. as a next step, don't think it's necessary to cover that for now in this PR).

**Comment by citizenrich** [comment]
Thanks @danfuterman and @MattyJ007. I can make my suggestion into a PR. Behind the scenes GitHub Actions run on Azure infrastructure VMs, so they could scale for the memory needs. But, I'll have a go at it on my local computer using https://github.com/nektos/act

**Pull request merged**